helm-charts/support/Chart.yaml

```
@@ -15,21 +15,21 @@ dependencies:
   - name: prometheus
     # NOTE: CHECK INSTRUCTIONS UNDER prometheus.server.command IN support/values.yaml
     # EACH TIME THIS VERSION IS BUMPED!
-    version: 25.8.2
+    version: 25.11.0
     repository: https://prometheus-community.github.io/helm-charts
 
   # Grafana for dashboarding of metrics.
   # https://github.com/grafana/helm-charts/tree/main/charts/grafana
   - name: grafana
-    version: 6.60.1
+    version: 7.3.0
```

Bumping to the latest version before 7

```suggestion
    version: 6.61.3
```